bundles/org.openhab.binding.openthermgateway/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -286,6 +286,69 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="unsuccessfulburnerstarts">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Unsuccessful Burner Starts</label>
+		<description>Unsuccessful burner starts</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="burnerstarts">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Burner Starts</label>
+		<description>Burner starts</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="chpumpstarts">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Central Heating Pump Starts</label>
+		<description>Central heating pump starts</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="dhwpvstarts">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Domestic Hot Water Pump/Valve Starts</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with max 20-25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Labels match those of OpenTherm messages. I could abbreviate Domestic Hot Water to DHW, but in my opinion that is not an improvement.

Reply:
I don't have any better idea. So, you can leave it as is.